tailwind.config.js

```diff
@@ -10,7 +10,11 @@ module.exports = {
   plugins: [require("daisyui")],
   daisyui: {
     themes: [
-      "light",
+      {
+        light: {
+          ...require("daisyui/src/theming/themes")["light"],
+          primary: "#55ACD8",        }
```

Comment:
Extending the theme was the best way to set the badge color. I intended to do this in the future for the colors that are often used for buttons and other often used components.